Framework/DataHandling/src/LoadILLSANS.cpp

```diff
@@ -256,12 +256,13 @@ LoadILLSANS::loadDataIntoWorkspaceFromMonitors(NeXus::NXEntry &firstEntry,
         positionsBinning.push_back(static_cast<double>(i));
 
       // Assign X
-      m_localWorkspace->dataX(firstIndex)
+      m_localWorkspace->mutableX(firstIndex)
           .assign(positionsBinning.begin(), positionsBinning.end());
```

Comment:
`m_localWorkspace->setBinEdges(firstIndex, vectorSize, LinearGenerator(0.0, 1.0));` and remove `positionsBinning`?